app/src/main/java/com/infomaniak/mail/utils/Extensions.kt

```diff
@@ -273,7 +273,10 @@ fun Fragment.safeNavigateToNewMessageActivity(args: Bundle? = null, currentClass
     if (canNavigate(currentClassName)) (activity as MainActivity).navigateToNewMessageActivity(args)
 }
 
-fun Fragment.navigateToThread(thread: Thread, mainViewModel: MainViewModel) = runCatchingRealm {
+fun Fragment.navigateToThread(
+    mainViewModel: MainViewModel,
+    thread: Thread,
+) = runCatchingRealm {
```

Comment:
Why ? 😭 
Keep it perfectly onelined

Reply:
Because it will be too long in a following PR.